source/platform_apple.cmake

```diff
@@ -0,0 +1,48 @@
+if( NOT APPLE)
+    message(FATAL_ERROR "Don't include this on non-macOS platforms.")
+endif()
```

Comment:
This looks very defensive. ;)

- Do `if(APPLE)` in `source/CMakeList.txt`, and remove this extra check
- Do only `include(...)` in `source/CMakeList.txt`, and put an `if(APPLE)` around all content of this file.

(I would go for the second.)

Reply:
Removed and renamed.
> (I would go for the second.)

That adds unnecessary indent (: